src/content/docs/en/guides/view-transitions.mdx

```diff
@@ -374,20 +374,6 @@ This has the effect that if you go back from the `/main` page, the browser will
 
 The `<ViewTransitions />` router also supports navigation triggered by form submissions. 
```

Comment:
```suggestion
The `<ViewTransitions />` router also animates navigations triggered by form submissions. 
```
You are right, the block below seems redundant to me as well. Its presence may also give the impression that there is something users will have to do make it work. I tried rewording this line hoping to make it clear that not only is it supported, it will work by default.